### PR TITLE
Deriv pos col set

### PR DESCRIPTION
**Pull request by ezewer**
Add rest endpoint derivsPositionCollateralSet
Required for tests https://github.com/bitfinexcom/bfx-api-mock-srv/pull/21

### CODE REVIEW

**Comment by JacobPlaster** [comment]
Same as other pr, please add integration test to `https://github.com/bitfinexcom/bfx-api-node-rest/blob/master/test/lib/rest-2-integration.js`

File: lib/rest2.js [hunk 1]
```diff
@@ -886,6 +886,24 @@ class RESTv2 {
...
+   * @param {string} symbol
+   * @param {number} collateral
+   * @param {Method?} cb - legacy callback
+   * @return {Promise} p
+   * @see https://docs.bitfinex.com/v2/reference#rest-auth-deriv-pos-collateral-set
+   */
+
+  derivsPositionCollateralSet (symbol = 'tBTCF0:USDF0', collateral, cb) {
+    const url = '/auth/w/deriv/collateral/set'
+    const isTrue = (res) => !!(res && res[0] && res[0][0])
```
Comment (JacobPlaster) [inline lib/rest2.js]: I might be being a little pedantic here but could we change the name of the this to something like `isRequestValid`?

Comment (ezewer) [reply lib/rest2.js]: sure!

File: lib/rest2.js [hunk 2]
```diff
@@ -886,6 +886,24 @@ class RESTv2 {
...
+   * Changes the collateral value of an active derivatives position for a certain pair.
+   *
+   * @param {string} symbol
+   * @param {number} collateral
+   * @param {Method?} cb - legacy callback
+   * @return {Promise} p
+   * @see https://docs.bitfinex.com/v2/reference#rest-auth-deriv-pos-collateral-set
+   */
+
+  derivsPositionCollateralSet (symbol = 'tBTCF0:USDF0', collateral, cb) {
```
Comment (JacobPlaster) [inline lib/rest2.js]: Could we remove the default assignment to `tBTCF0:USDF`? I think its best to let the request fail and return an error response then them accidentally update the wrong derivative position

**Pull request merged**